### ISSUE TITLE
The ability to use objects for the select editor

### ISSUE THREAD

**Issue by moetun88** [olifolkerd/tabulator#2122]
**Is your feature request related to a problem? Please describe.**
If the select editor values are based on a json object, I can stringify the values while defining them via editorParams which works beautifully.  On selection they are destringfied using the cellEdited callback before updating the backing json object.

The problem is the intialvalue is retrieved (on line 550 of edit.js) from cell.getValue() which returns the non stringified json object. There is no way I can tell to intercept and stringify the object.

**Describe the solution you'd like**
Have callbacks to pre-process or inject the initial value.

In summary for the select editor
editorParams => stringify object values (works)
celledited => destringify object values (works)
select: function => ??? - need the ability to destringify the value from cell.getValue()

**Comment by olifolkerd** [comment]
Hey @moetun88 

Im afraid i don't understand your issue, why do they need to be stringified at all? why cant you just pass in the object?

Cheers

Oli :)
